app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginViewModel.java

```diff
@@ -43,8 +42,8 @@ public LoginViewModel(AuthService loginModel, HostSelectionInterceptor intercept
 
     //for logging into the app
     public void login() {
+        progress.setValue(false);
```

Comment:
Why posting false?

Reply:
So what change should I be doing here?

Reply:
true

Reply:
Even so, you should be doing that through doOnSubscribe

Reply:
progress has already been set to true in doOnSubscribe